### PR TITLE
use python script instead for auto generating binding codes

### PR DESCRIPTION
**Pull request by lihex**

### CODE REVIEW

File: tools/tolua/genbindings.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+    conf_ini_file = os.path.abspath(os.path.join(os.path.dirname(__file__), 'userconf.ini')) 
+
+    print 'generating userconf.ini...'
+    with open(conf_ini_file, 'w') as configfile:
+      config.write(configfile)
+
+
+    # set proper environment variables
+    if 'linux' in platform or platform == 'darwin':
```
Comment (dumganhar) [inline tools/tolua/genbindings.py:116]: Why not just use `platform == 'linux'`?

Comment (lihex) [inline tools/tolua/genbindings.py:116]: since sys.platform returns 'linux2' not 'linux' on linux

Comment (dumganhar) [inline tools/tolua/genbindings.py:116]: ok

File: tools/tojs/genbindings.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    ndk_root = re.sub(r"\"", "", ndk_root)
+
+    platform = sys.platform
+    if platform == 'win32':
+        cur_platform = 'windows'
+    elif platform == 'darwin':
+        cur_platform = platform
+    elif 'linux' in platform:
+        cur_platform = 'linux'
+    else:
```
Comment (litefeel) [inline tools/tojs/genbindings.py:74]: no cygwin?

Comment (dumganhar) [inline tools/tojs/genbindings.py:74]: Yep, it isn't needed.

**Pull request merged**